libclamav_rust/build.rs

```diff
@@ -141,7 +145,9 @@ fn execute_bindgen() -> Result<(), &'static str> {
         // are probably not the same when generated as when compiled.
         .layout_tests(false)
         // Enable bindgen to find generated headers in the build directory, too.
-        .clang_arg(build_include_path);
+        .clang_arg(build_include_path)
+        // Hack to include 3rd party C libary headers
+        .clang_arg("-I/home/micah/.mussels/install/host-static/include");
```

Comment:
What is the plan for removing this hardcoded path?

Reply:
Ooops!  Good catch!  This shouldn't stay in the PR. I will remove it.

I needed it to make `MAINTAINER_MODE` work.  It is currently broken.  The real fix (later, see CLAM-2451) will be to provide 3rd party library include paths from cmake to build.rs `MAINTEINER_MODE` is enabled so that bindgen can find them.